Formula/watchman.rb

```diff
@@ -41,11 +47,15 @@ class Watchman < Formula
   def install
     # Fix build failure on Linux. Borrowed from Fedora:
     # https://src.fedoraproject.org/rpms/watchman/blob/rawhide/f/watchman.spec#_70
-    inreplace "CMakeLists.txt", /^t_test/, "# t_test" if OS.linux?
+    inreplace "CMakeLists.txt", /^t_test/, "#t_test" if OS.linux?
 
+    # NOTE: Setting `BUILD_SHARED_LIBS=ON` will generate DSOs for Eden libraries.
+    #       These libraries are not part of any install targets and have the wrong
+    #       RPATHs configured, so will need to be installed and relocated manually
+    #       if they are built as shared libraries. They're not used by any other
+    #       formulae, so let's link them statically instead. This is done by default.
     system "cmake", "-S", ".", "-B", "build",
-                    "-DBUILD_SHARED_LIBS=ON",
-                    "-DENABLE_EDEN_SUPPORT=OFF",
+                    "-DENABLE_EDEN_SUPPORT=#{OS.mac?}",
```

Comment:
Does this convert to a useful string?

Reply:
Yes.
```
[1] brew(main)> true.to_s
=> "true"
[2] brew(main)> false.to_s
=> "false"
```
https://cmake.org/cmake/help/latest/command/if.html#basic-expressions